docs/OTP2-MigrationGuide.md

```diff
@@ -25,22 +25,50 @@ you want into a single instance of OTP2.
 
 ## Build config
 
+New parameters:
+
+ - `transitServiceStart` Limit the import of transit services to the given *start* date. Default: `-P1Y`
+ - `transitServiceEnd` Limit the import of transit services to the given *end* date. *Inclusive*. Default: `P3Y`
+
+
 These properties changed names from:
 
  - `htmlAnnotations` to `dataImportReport`
  - `maxHtmlAnnotationsPerFile` to `maxDataImportIssuesPerFile`
  - `boardTimes` to `routingDefaults.boardSlackByMode`
  - `alightTimes` to `routingDefaults.alightSlackByMode`
  
-These parameters is no longer supported:
-
- - `stopClusterMode` - TODO OTP2 Why? Old options: `proximity`, `parentStation`
+These parameters are no longer supported:
+
+ - `stopClusterMode` OTP2 link stops and stations based on the GTFS/NeTEx input. There is no
+         automatic linking any more. If this is needed, feel free to port the OTP1 logic into 
+         a Sandbox feature. The OTP1 options are: `proximity`, `parentStation`.
+ - `parentStopLinking` OTP2 link stops and stations based on the GTFS/NeTEx input, but there is no 
+         such thing in the street graph, only stops exist in the street graph. When OTP2 generate
+         transfers based on the street graph generated from OSM data. If no OSM data is provided,
+         then we would like to use "line-of-sight" transfers. See issue 
+         [#3204](https://github.com/opentripplanner/OpenTripPlanner/issues/3204).  
+ - `stationTransfers` This functionality is not ported to OTP2. If needed it should be a sandbox
+         feature. 
 
  
 ## Router config
+Se the [Router Configuration](Configuration.md#router-configuration) for a description of the 
+new and continued routing parameters.
+
+New parameters:
+
+ - `streetRoutingTimeout` Maximum time limit for street route queries. Replace the old `timeout`.
```

Comment:
It could be good to mention the units.

Reply:
This is the migration guide, not the documentation of the feature - people should look at the configuration.md. I have tried to give a small hint, but let out most of the documentation, so it is possible to get a "bird-view" of the changes.